.repo-metadata.json

```diff
@@ -1,11 +1,12 @@
 {
-  "name": "google-auth",
-  "name_pretty": "Google Auth Python Library",
-  "client_documentation": "https://googleapis.dev/python/google-auth/latest",
-  "issue_tracker": "https://github.com/googleapis/google-auth-library-python/issues",
-  "release_level": "ga",
-  "language": "python",
-  "library_type": "AUTH",
-  "repo": "googleapis/google-auth-library-python",
-  "distribution_name": "google-auth"
+    "name": "google-auth",
+    "name_pretty": "Google Auth Python Library",
+    "client_documentation": "https://cloud.google.com/python/docs/reference/google-auth/latest",
+    "issue_tracker": "https://github.com/googleapis/google-auth-library-python/issues",
+    "release_level": "stable",
+    "language": "python",
+    "library_type": "AUTH",
+    "repo": "googleapis/google-auth-library-python",
+    "distribution_name": "google-auth",
+    "api_shortname": "google-auth"
```

Comment:
Ditto similar questions I've left on other repos - is the `api_shortname` required for all repos (even non-GAPICs)? Is there a standard pattern for what it should be when there is no Google API?

Reply:
Fixed